### PR TITLE
Move transaction serialization in correct directory

### PR DESCRIPTION
**Pull request by sprohaszka-ledger**
### ✅ Checklist

- [ ] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Set together all CoinFmk serialization functions.
Create a dedicated "formatters" file.

### ❓ Context

- **JIRA or GitHub link**: 

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: libs/coin-framework/package.json [hunk 1]
```diff
@@ -71,10 +61,18 @@
       "require": "./lib/nft/*.js",
       "default": "./lib-es/nft/*.js"
     },
+    "./serialization": {
+      "require": "./lib/serialization/index.js",
```
Comment (lvndry) [inline libs/coin-framework/package.json:65]: Never understood why it was required to have a `lib` and `lib-es` do know why ? It feels like it's more complicated than it should be

File: libs/coin-framework/src/formatters.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  str +=
+    "\n" +
+    `errors: ${Object.entries(errors)
+      .map(([key, error]) => `${key} ${formatErrorSmall(error)}`)
+      .join(", ")}`;
+
+  str +=
+    "\n" +
+    `errors: ${Object.entries(warnings)
```
Comment (lvndry) [inline libs/coin-framework/src/formatters.ts:44]:
```suggestion
    `warnings: ${Object.entries(warnings)
```

File: libs/coin-framework/package.json (hunk 1 above)
Comment (sprohaszka-ledger) [reply libs/coin-framework/package.json:65]: It depends if you need `commonjs` or `esm` module.

File: libs/coin-framework/src/formatters.ts (hunk 2 above)
Comment (lvndry) [reply libs/coin-framework/src/formatters.ts:44]: I guess it's only formatting of a string it shouldn't have side effects right ?

**Review by lvndry — APPROVED** [review]

**Review by hedi-edelbloute — APPROVED** [review]

**Pull request merged**